src/cljs/rems/actions.cljs

```diff
@@ -51,31 +55,42 @@
    [show-throughput-times-button]])
 
 (defn actions-page []
-  [:div
-   [document-title (text :t.navigation/actions)]
-   [flash-message/component :top]
-   [:div.spaced-sections
-    [collapsible/component
-     {:id "todo-applications"
-      :open? true
-      :title (text :t.actions/todo-applications)
-      :collapse [:<>
-                 [search/application-search-field {:id "todo-search"
-                                                   :on-search #(rf/dispatch [::todo-applications {:query %}])
-                                                   :searching? @(rf/subscribe [::todo-applications :searching?])}]
-                 [application-list/component {:applications ::todo-applications
-                                              :hidden-columns #{:state :created}
-                                              :default-sort-column :last-activity
-                                              :default-sort-order :desc}]]}]
-    [collapsible/component
-     {:id "handled-applications"
-      :on-open #(rf/dispatch [::handled-applications])
-      :title (text :t.actions/handled-applications)
-      :collapse [:<>
-                 [search/application-search-field {:id "handled-search"
-                                                   :on-search #(rf/dispatch [::handled-applications {:query %}])
-                                                   :searching? @(rf/subscribe [::handled-applications :searching?])}]
-                 [application-list/component {:applications ::handled-applications
-                                              :hidden-columns #{:todo :created :submitted}
-                                              :default-sort-column :last-activity
-                                              :default-sort-order :desc}]]}]]])
+  (let [config @(rf/subscribe [:rems.config/config])]
+    [:div
+     [document-title (text :t.navigation/actions)]
+     [flash-message/component :top]
+     (when (:enable-ega config)
+       (if-let [ega-api-key-expiration-date (get-in @(rf/subscribe [::user-settings])
+                                                    [:ega :api-key-expiration-date])]
+         (when (time-core/after? (time-core/now) ega-api-key-expiration-date)
```

Comment:
how about warning about impending expiration?

Reply:
Good question. There are some other considerations of how the edge cases should actually be handled. Like warning mail. I think I will gather some of these to the ADR "unfinished business" part and we can solve them when someone actually takes this into use and we can talk about the process they wish for this.

Reply:
This is now issue https://github.com/CSCfi/rems/issues/2686